dts/bindings/i2c/synopsys,dw-i2c.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: DesignWare SPI node description
+id: synopsys,dw-i2c
```

Comment:
We should use/conform to the linux binding Documentation/devicetree/bindings/i2c/i2c-designware.txt (for things like compat, etc)